.github/workflows/check_vue_tsc.yml

```diff
@@ -0,0 +1,13 @@
+name: check-vue-tsc
+on: 
+  pull_request
+
+jobs:
+  vue-tsc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 18
+      - uses: keita-hino/vue-tsc-action@main
```

Comment:
@sonya0504 
Using @main for actions can be risky as the main branch may receive updates that break backward compatibility. Instead, consider using a specific version or commit hash.

Reply:
I removed this `vue-tsc-action` and wrote an action from scratch